cmd/defaults.go

```diff
@@ -88,7 +168,14 @@ func LoadOrInit(github string, toolHomeEnvName string) (*ToolConfig, error) {
 		if err := config.Save(); err != nil {
 			return nil, err
 		}
+	} else {
```

Comment:
not sure how to do this with NGS, right now i just use tool home, not a comment for this code, but seeing what you think